pkg/spec/config_linux_cgo.go

```diff
@@ -0,0 +1,34 @@
+// +build linux,seccomp
```

Comment:
Should the flag here be cgo, not seccomp?

Reply:
I've reused seccomp since we already have a build flag, but thinking more of it I think cgo is better.  I'll change it